docs/Running-Mastodon/Production-guide.md

```diff
@@ -34,9 +34,17 @@ server {
   keepalive_timeout    70;
   sendfile             on;
   client_max_body_size 0;
-  gzip off;
 
-  root /home/mastodon/live/public;
+
+  gzip on;
```

Comment:
Yeah, it's work fine on https://mastodon.cloud,

Better performance with Gzip enabled.  
https://gtmetrix.com/reports/mastodon.cloud/FBZCuj81